### PR TITLE
Add a page with external guides

### LINKED ISSUE #428: Rethink blog
I like Medium for authoring and its "social" aspects, but I really dislike how difficult it makes finding older blog posts. And we had a lot of great old posts that discuss architecture and deployment modes of Jaeger, which I think people are completely missing. I think we need to rethink how we maintain the blog posts. Some options:

* host all posts in this repo
  * pros:
    * authoring in Markdown is just as easy
  * cons: 
    * inserting images is somewhat harder
    * lacking "social" aspects, like following specific authors, or even reading posts by a given author (possible by will require some Hugo templating changes)
* continue hosting on Medium, but maintain an index on the website (more extensive than the carousel on the home page).
  * we could build index automatically similar to the carousel
* convert some of the blog posts into official documentation (e.g. discussion on multi-tenancy, etc.)
  * this could also take form of an extended FAQ section with direct links to the blog posts on Medium

### PR DESCRIPTION
**Pull request by yurishkuro**
Resolves #428

### CODE REVIEW

**Review by albertteoh — APPROVED** [review]

**Comment by yurishkuro** [comment]
thanks for review @albertteoh

File: content/docs/1.29/external-guides.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+* [Protecting Jaeger UI with an OAuth sidecar Proxy](https://medium.com/jaegertracing/protecting-jaeger-ui-with-an-oauth-sidecar-proxy-34205cca4bb1) by Juraci Paixão Kröhling
+* [Jaeger and multi-tenancy](https://medium.com/jaegertracing/jaeger-and-multitenancy-99dfa1d49dc0) by Juraci Paixão Kröhling
+* [Deployment strategies for the Jaeger Agent](https://medium.com/jaegertracing/deployment-strategies-for-the-jaeger-agent-1d6f91796d09) by Juraci Paixão Kröhling
+* [How to deploy Jaeger on AWS: a comprehensive step-by-step guide](https://www.aspecto.io/blog/how-to-deploy-jaeger-on-aws-a-comprehensive-step-by-step-guide/) by Tom Zach
+
+## Jaeger Performance Tuning
+
+* [How to maximize span ingestion while limiting writes per second to Scylla with Jaeger](https://medium.com/jaegertracing/how-to-maximize-span-ingestion-while-limiting-writes-per-second-to-scylla-with-jaeger-3bcda5608841) by Joe Elliott
+
+## Troublshooting
```
Comment (jpkrohling) [inline content/docs/1.29/external-guides.md:36]: s/Troublshooting/Troubleshooting

File: content/docs/next-release/external-guides.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+* [Protecting Jaeger UI with an OAuth sidecar Proxy](https://medium.com/jaegertracing/protecting-jaeger-ui-with-an-oauth-sidecar-proxy-34205cca4bb1) by Juraci Paixão Kröhling
+* [Jaeger and multi-tenancy](https://medium.com/jaegertracing/jaeger-and-multitenancy-99dfa1d49dc0) by Juraci Paixão Kröhling
+* [Deployment strategies for the Jaeger Agent](https://medium.com/jaegertracing/deployment-strategies-for-the-jaeger-agent-1d6f91796d09) by Juraci Paixão Kröhling
+* [How to deploy Jaeger on AWS: a comprehensive step-by-step guide](https://www.aspecto.io/blog/how-to-deploy-jaeger-on-aws-a-comprehensive-step-by-step-guide/) by Tom Zach
+
+## Jaeger Performance Tuning
+
+* [How to maximize span ingestion while limiting writes per second to Scylla with Jaeger](https://medium.com/jaegertracing/how-to-maximize-span-ingestion-while-limiting-writes-per-second-to-scylla-with-jaeger-3bcda5608841) by Joe Elliott
+
+## Troublshooting
```
Comment (jpkrohling) [inline content/docs/next-release/external-guides.md:36]: s/Troublshooting/Troubleshooting

**Comment by albertteoh** [comment]
Good catch @jpkrohling! Fixed in #545.

**Pull request merged**